EIPS/eip-2348.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: 2348
+title: Validated EVM Contracts
+author: Danno Ferrin (@shemnon)
+discussions-to: https://ethereum-magicians.org/t/eip-2348-validated-evm-contracts/3756
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-11-01
+requires: 1702
+---
+
+<!--
+HTML checker won't let me reference unpublished EIPs, namely 2327, 1707, 1712
+requires: 1702, 2327
+replaces: 615 (in part), 1707 (unpublished, abandoned), 1712 (unpublished)
```

Comment:
Btw strictly speaking the `replaces` field can only reference Final (aka. deployed) EIPs.